staging/src/k8s.io/apimachinery/pkg/util/json/json.go

```diff
@@ -46,6 +46,15 @@ func Unmarshal(data []byte, v interface{}) error {
 	return kjson.UnmarshalCaseSensitivePreserveInts(data, v)
 }
 
+// UnmarshalStrict performs a strict unmarshal
+// performing additional checks to ensure the data contains
+// no duplicate or unknown fields and returning those separately
+// as strictErrors (while succeeding with the unmarshal into v)
+// as long as there are no fatal errors.
+func UnmarshalStrict(data []byte, v interface{}) (strictErrors []error, err error) {
+	return kjson.UnmarshalStrict(data, v)
+}
```

Comment:
is there a reason to increase the surface area of this package instead of switching callers to call `kjson.UnmarshalStrict` directly?

Reply:
nope, I think I just did it for consistency with the other `Unmarshal` call being made here. Removed it.